rllib/agents/ddpg/ddpg_policy.py

```diff
@@ -3,7 +3,7 @@
 
 import ray
 import ray.experimental.tf_utils
-from ray.rllib.agents.dqn.dqn_policy import _postprocess_dqn
+from ray.rllib.agents.dqn.dqn_policy import postprocess_fn
```

Comment:
Consider `postprocess_nstep_and_prio` here to having an undescriptive function name.

Reply:
Changed.